lib/rubocop/cop/rspec/change_by_zero.rb

```diff
@@ -29,6 +31,8 @@ module RSpec
       #   # good
       #   expect { run }.not_to change(Foo, :bar)
       #   expect { run }.not_to change { Foo.bar }
+      #   expect { run }.not_to a_block_changing(Foo, :bar)
+      #   expect { run }.not_to changing(Foo, :bar)
```

Comment:
I would omit these two lines from the documentation.

Reply:
I see. I removed it 👍🏻